src/index.html

```diff
@@ -12,36 +12,46 @@
     <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
```

Comment:
- [ ] [OPTIONAL] The screenshot you are using in the README is for another project, kindly change it before you merge the changes. this is highly recommended 

![Screenshot_20220616_230139](https://user-images.githubusercontent.com/25227022/174179015-ce778a17-3029-44aa-b248-b965895c460e.png)